tests/test_qiskit_device.py

```diff
@@ -140,8 +141,10 @@ def test_calls_to_reset(self, n_tapes, mocker, device):
 
         assert spy.call_count == n_tapes
 
-    def test_result(self, device, tol):
+    def test_result_legacy(self, device, tol):
         """Tests that the result has the correct shape and entry types."""
+        # TODO: remove once the legacy return system is activated.
```

Comment:
```suggestion
        # TODO: remove once the legacy return system is removed.
```

Reply:
Also, the failure of this test once the legacy system is removed will be a pretty clear indication this needs to be removed.